v3/integrations/nrpgx5/nrpgx5.go

```diff
@@ -83,7 +84,8 @@ const (
 
 func NewTracer() *Tracer {
 	return &Tracer{
-		ParseQuery: sqlparse.ParseQuery,
+		ParseQuery:          sqlparse.ParseQuery,
+		SendQueryParameters: true,
```

Comment:
could we default this to false? I don't want customers to feel that their sensitive data was exposed unexpectedly, and so I'd like to preserve the default behavior.

Reply:
Just to clarify, the default behavior before this PR was to always send these parameters. This change preserves that for backwards compatibility.

Reply:
Oh, gotcha. In that case this is fine then.